tests/js/fixtures/sentryAppComponent.js

```diff
@@ -3,7 +3,18 @@ export function SentryAppComponent(params = {}) {
     uuid: 'ed517da4-a324-44c0-aeea-1894cd9923fb',
     type: 'issue-link',
     schema: {
-      link: {required_fields: [{type: 'text', name: 'a', label: 'A'}]},
+      link: {
+        required_fields: [
+          {type: 'text', name: 'a', label: 'A', default: 'issue.title'},
+          {type: 'textarea', name: 'c', label: 'C', default: 'issue.description'},
+          {
+            type: 'select',
+            name: 'numbers',
+            label: 'Numbers',
+            options: [{label: 'one', value: 1}, {label: 'two', value: 2, default: true}],
```

Comment:
We don't currently have the schema defined this way. If you're specifying `options` statically, it expects an array of arrays.

We can change this, though, to be consistent with the response format we expect when making a request to retrieve options (and how you have it specified here).

But currently, this example schema isn't valid.